components/link/link.js

```diff
@@ -125,6 +141,7 @@ class Link extends FocusMixin(LitElement) {
 		this.main = false;
 		this.small = false;
 		this.lines = 0;
+		this.showOpenInNewWindow = false;
```

Comment:
Doesn't appear to be used. Probably mean to do this?

```suggestion
		this.newWindow = false;
```